lib/pharos/salt_minion.rb

```diff
@@ -30,8 +30,9 @@ def roles?
     end
 
     # Assign role to this minion.
-    def assign_role(role:)
+    def assign_role(role)
```

Comment:
Same as before. At a first glance it looks like a good idea to have this as a keyword argument (let's be consistent, we are using keyword arguments in lots of places already).